modules/ossm-troubleshooting-injection.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-injection_{context}"]
+= Troubleshooting sidecar injection
+
+Unlike how the upstream Istio community configures proxy sidecars using namespace labels, {ProductName} does not automatically inject proxy sidecars to pods.  You must opt in to sidecar injection.
```

Comment:
This says OSSM is unlike the upstream, but the first heading mentions Istio, which might be confusing. I also notice that the paragraphs underneath both the istio and jaeger sidecar injection are broadly similar. Is it possible to just combine those into one paragraph. So instead of Troubleshooting sidecar injection with two subheads, 

= Troubleshooting sidecar injection

Check to see if automatic injection is enabled in the Deployment for your application.  If automatic injection for the Envoy proxy is enabled, there should be a `sidecar.istio.io/inject:"true"` and `sidecar.jaegertracing.io/inject:"true"` annotation in the Deployment resource under `spec.template.metadata.annotations`.

I had one more thought here. This module has an explanation for what a sidecar injection is and how to fix it, but it doesn't mention the symptom. Many troubleshooting topics start with the symptom and then reveal the cure. What is the symptom? How do I know I have a sidecar injection problem?